iacrcc/iacrdoc.tex

```diff
@@ -138,7 +145,7 @@ \section{Invocation and usage}
 Before submitting your final version, please make sure that it compiles
 properly with the \texttt{[version=final]} option
 and check that the author names and affiliations are
-correct.
+correct and a text version of the abstract is provided.
```

Comment:
suggest ending this sentence with 
... in the \texttt{textabstract} environment.

Reply:
This has been added.